### PR TITLE
Update es.po

### PR DESCRIPTION
**Pull request by alainm23**
Update spanish traslate

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -1,271 +1,337 @@
...
 "Content-Type: text/plain; charset=UTF-8\n"
-"Plural-Forms: nplurals=2; plural=n != 1;\n"
-"Project-Id-Version: playmymusic\n"
-"PO-Revision-Date: \n"
-"Last-Translator: Adolfo Jayme Barrientos <fitojb@ubuntu.com>\n"
-"Language-Team: \n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.7.1\n"
+"Last-Translator: Adolfo Jayme Barrientos <fitojb@ubuntu.com>\n"
+"Last-Translator: Alain M. <alain23@protonmail.com>\n"
```
Comment (fitojb) [inline po/es.po]: The `Last-Translator` header is duplicated, which makes for an invalid gettext file.

File: po/es.po [hunk 2]
```diff
@@ -1,271 +1,337 @@
...
-"X-Generator: Poedit 2.0.4\n"
 
-#: ../src/Widgets/Album.vala:100 ../src/Widgets/Artist.vala:85
-#: ../src/Widgets/Views/AlbumView.vala:86
-#: ../src/Widgets/Views/AudioCDView.vala:118
-msgid "Set new Cover…"
-msgstr "Definir carátula nueva…"
+#: ../src/Dialogs/AlbumEditor.vala:82 ../src/Dialogs/ArtistEditor.vala:81
+msgid "Click to choose a new cover…"
+msgstr "Añadir caratula…"
```
Comment (fitojb) [inline po/es.po]: _Carátula_ is a proparoxytone word. In Spanish, **all** proparoxytones take a mandatory accent mark.

File: po/es.po [hunk 3]
```diff
@@ -1,271 +1,337 @@
...
+msgid "Change Music Folder…"
+msgstr "Cambiar carpeta de música…"
 
-#: ../src/Widgets/Views/RadiosView.vala:226
-#: ../src/Widgets/Views/PlaylistsView.vala:158
-msgid "Add"
-msgstr "Añadir"
+#: ../src/MainWindow.vala:395
+msgid "Import Music…"
+msgstr "Importar musica…"
```
Comment (fitojb) [inline po/es.po]: _Música_ also missing its accent.

File: po/es.po [hunk 4]
```diff
@@ -1,271 +1,337 @@
...
+#: ../src/MainWindow.vala:414
+msgid "Preferences"
+msgstr "Preferencias"
 
-#: ../src/Widgets/Views/PlaylistView.vala:124
-msgid "Remove Playlist"
-msgstr "Quitar lista"
+#: ../src/MainWindow.vala:434
+msgid "Search Music"
+msgstr "Buscar Musica"
```
Comment (fitojb) [inline po/es.po]: In Spanish (and all other Romance languages), calquing English’s Title Case is incorrect.

File: po/es.po [hunk 5]
```diff
@@ -1,271 +1,337 @@
...
+#: ../src/MainWindow.vala:483
+msgid "Repeat"
+msgstr "Repetir"
 
-#: ../src/Widgets/Views/PlaylistsView.vala:178
-msgid "Add Playlist"
-msgstr "Añadir lista"
+#: ../src/MainWindow.vala:547
+msgid "Albums"
+msgstr "Albunes"
```
Comment (fitojb) [inline po/es.po]: The word is _álbumes_, with an accent mark and an _m_.

File: po/es.po [hunk 6]
```diff
@@ -1,271 +1,337 @@
...
-#: ../src/Widgets/Track.vala:61
-msgid ""
-"File couldn't be found\n"
-"%s"
-msgstr ""
-"No se pudo encontrar el archivo\n"
-"%s"
+#: ../src/MainWindow.vala:556
+msgid "Playlists"
+msgstr "Playlist"
```
Comment (fitojb) [inline po/es.po]: I don’t know why you chose not to translate the word _playlist_, which as a compound word has a really obvious Spanish version: _lista de reproducción_ or simply _lista_.

File: po/es.po [hunk 7]
```diff
@@ -1,271 +1,337 @@
...
 
-#: ../src/Widgets/Track.vala:104
-msgid "Remove from Playlist"
-msgstr "Quitar de lista"
+#: ../src/MainWindow.vala:561
+msgid "Radio Stations"
+msgstr "Estacion de radio"
```
Comment (fitojb) [inline po/es.po]: _Estación_ takes an accent because it’s an oxytone word ending in _n_, _s_ or vowel.

File: po/es.po [hunk 8]
```diff
@@ -1,271 +1,337 @@
...
+#: ../src/Services/LibraryManager.vala:345
+#: ../src/Services/LibraryManager.vala:366
 msgid "_Open"
 msgstr "_Abrir"
 
-#: ../src/Services/LibraryManager.vala:252
+#: ../src/Services/LibraryManager.vala:348
 msgid "Images"
-msgstr "Imágenes"
+msgstr "Imagenes"
```
Comment (fitojb) [inline po/es.po]: WTF

**Comment by artemanufrij** [comment]
@fitojb thanks for your feedback and you translation PRs. I can't check spanish translations because I don't speak it. :man_shrugging: 
I have to trust the users...

**Pull request merged**